src/sentry/issues/ingest.py

```diff
@@ -39,7 +41,11 @@
 def save_issue_occurrence(
     occurrence_data: IssueOccurrenceData, event: Event
 ) -> Tuple[IssueOccurrence, Optional[GroupInfo]]:
-    process_occurrence_data(occurrence_data)
+    # Do not hash fingerprints for performance issues because they're already
+    # hashed in save_aggregate_performance
+    # This check should be removed once perf issues are created through the platform
+    if can_create_group(occurrence_data, event.project):
```

Comment:
`IssueOccurrenceData` is a dict, we'll need to handle that type in `can_create_group` as well